python/friesian/example/multi_task/run_multi_task.py

```diff
@@ -0,0 +1,250 @@
+        sc = init_orca_context("standalone", master=args.master,
+                               cores=args.executor_cores, num_nodes=args.num_executors,
+                               memory=args.executor_memory,
+                               driver_cores=args.driver_cores,
+                               driver_memory=args.driver_memory)
+    elif args.cluster_mode == "yarn":  # For Hadoop/YARN cluster
+        conf = {"spark.executor.memoryOverhead": "130g",
+                "spark.network.timeout": "10000000",
+                "spark.sql.broadcastTimeout": "7200",
+                "spark.sql.shuffle.partitions": "2000",
+                "spark.locality.wait": "0s",
+                "spark.sql.crossJoin.enabled": "true",
+                "spark.task.cpus": "1",
+                "spark.executor.heartbeatInterval": "200s",
+                "spark.driver.maxResultSize": "40G",
+                "spark.eventLog.enabled": "true",
+                "spark.app.name": "recsys-demo-train"}
+        sc = init_orca_context(cluster_mode="yarn", cores=args.executor_cores,
+                               num_nodes=args.num_executors, memory=args.executor_memory,
+                               driver_cores=args.driver_cores, driver_memory=args.driver_memory,
+                               conf=conf, object_store_memory="80g",
+                               env={"KMP_BLOCKTIME": "1",
+                                    "KMP_AFFINITY": "granularity=fine,compact,1,0",
+                                    "OMP_NUM_THREADS": "28"})
+    elif args.cluster_mode == "spark-submit":
+        sc = init_orca_context("spark-submit")
+    else:
+        ArgumentError(False,
+                      "cluster_mode should be one of 'local', 'yarn', 'standalone' and"
+                      " 'spark-submit', but got " + args.cluster_mode)
+    cat_cols_ = [
+        'user_id',
+        'article_id',
+        'net_status',
+        'exop_position',
+        'device',
+        'city',
+        'age',
+        'gender',
+        'cat_1',
+    ]
+    continuous_cols_ = ['img_num']
+    feature_max_idx_ = {'user_id': 40000, 'article_id': 200000, 'net_status': 1004,
+                        'exop_position': 2000, 'device': 2000,
+                        'city': 1379, 'age': 1005, 'gender': 1003, 'cat_1': 1038}
+
+    if args.do_train:
+        train_tbl = FeatureTable.read_parquet(args.train_data_path)
+        valid_tbl = FeatureTable.read_parquet(args.test_data_path)
+        train_multi_task(train_tbl, valid_tbl, args.model_save_path,
+                         args.model_type, cat_cols_, continuous_cols_,
+                         feature_max_idx_)
+    elif args.do_test:
+        valid_tbl = FeatureTable.read_parquet(args.test_data_path)
+        test_multi_task(valid_tbl, args.model_save_path, args.model_type,
+                        cat_cols_, continuous_cols_, feature_max_idx_)
+    else:
+        print("Need to choose whether to do_train or do_test.")
```

Comment:
can we just remove the flat, first do train and then test in the same time?

Reply:
no problem